tools/SetupFlow/DevHome.SetupFlow.ElevatedComponent/Tasks/ElevatedConfigurationTask.cs

```diff
@@ -34,12 +34,19 @@ public IAsyncOperation<ElevatedConfigureTaskResult> ApplyConfiguration(StorageFi
                 taskResult.TaskAttempted = true;
                 taskResult.TaskSucceeded = result.Succeeded;
                 taskResult.RebootRequired = result.RequiresReboot;
-                taskResult.UnitResults = result.Result.UnitResults.Select(unitResult => new ElevatedConfigureUnitTaskResult
+                taskResult.UnitResults = result.Result.UnitResults.Select(unitResult =>
                 {
-                    UnitName = unitResult.Unit.UnitName,
-                    Intent = unitResult.Unit.Intent.ToString(),
-                    IsSkipped = unitResult.State == ConfigurationUnitState.Skipped,
-                    HResult = unitResult.ResultInformation?.ResultCode?.HResult ?? HRESULT.S_OK,
+                    object descriptionObj;
+                    unitResult.Unit.Directives.TryGetValue("description", out descriptionObj);

```

Comment:
WinGet does not surface the description today. Related issue:
https://github.com/microsoft/winget-cli/issues/3388

Reply:
Yeah, I didn't find any other way to get access to this description. Because of this I added this code to deal with `null` values: `Description = descriptionObj?.ToString() ?? string.Empty`.

To me use this description, if available, is a huge UX improvement and from my point of view should be a _must_.

Reply:
I am slightly concerned about showing a description which may not be the appropriate locale (not sure if winget config even allows localizing that...)
But it's definitely better than the also-not-localized IDs we use now.